packages/dd-trace/src/appsec/standalone.js

```diff
@@ -1,12 +1,57 @@
 'use strict'
 
 const { channel } = require('dc-polyfill')
-const { APM_TRACING_ENABLED_KEY, APPSEC_PROPAGATION_KEY } = require('../constants')
+const { USER_KEEP, AUTO_KEEP, AUTO_REJECT } = require('../../../../ext/priority')
+const { MANUAL_KEEP } = require('../../../../ext/tags')
+const PrioritySampler = require('../priority_sampler')
+const RateLimiter = require('../rate_limiter')
+const TraceState = require('../opentracing/propagation/tracestate')
+const { hasOwn } = require('../util')
+
+const {
+  APM_TRACING_ENABLED_KEY,
+  APPSEC_PROPAGATION_KEY
+} = require('../constants')
```

Comment:
not sure why those constants are multiline or separate from the rest of the requires ? since we also have other constants like priority and tags ?